src/django_otp_webauthn/helpers.py

```diff
@@ -193,12 +195,12 @@ def get_credential_display_name(self, user: AbstractUser) -> str:
         User.get_username().
         """
 
-        if user.get_full_name():
+        if hasattr(user, "get_full_name"):
```

Comment:
This appeared to be the only place actually depending on functionality which is defined only on `AbstractUser`. This should safely preserve the previous functionality while also providing the freedom to specify `AbstractBaseUser`.

If users need something drastically different, it looks like you've already got the functionality for developers to specify their own helpers class which could provide whatever implementation they need.